picard/util/tags.py

```diff
@@ -166,652 +344,1118 @@ def names(self, selector=None):
+        longdesc=N_('The names of the "*Mixed By*" engineers associated with the track.'),
     ),
     TagVar(
         'mood',
         shortdesc=N_('Mood'),
+        longdesc=N_('The mood associated with the track.'),
+        is_from_mb=False,
     ),
     TagVar(
         'movement',
         shortdesc=N_('Movement'),
+        longdesc=N_('Name of the movement (e.g.: "Andante con moto").'),
+        is_from_mb=False,
     ),
     TagVar(
         'movementnumber',
         shortdesc=N_('Movement Number'),
+        longdesc=N_(
+            'Movement number in Arabic numerals (e.g.: "2"). Players explicitly supporting this tag will often '
+            'display it in Roman numerals (e.g.: "II").'
+        ),
+        is_from_mb=False,
     ),
     TagVar(
         'movementtotal',
         shortdesc=N_('Movement Count'),
+        longdesc=N_('Total number of movements in the work (e.g.: "4").'),
+        is_from_mb=False,
     ),
     TagVar(
         'multiartist',
         shortdesc=N_('FIXME:multiartist'),
+        longdesc=N_(
+            'Set to 1 if not all of the tracks on the album have the same primary artist, otherwise empty.'
+        ),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'musicbrainz_albumartistid',
         shortdesc=N_('MusicBrainz Release Artist Id'),
+        longdesc=N_('A multi-value tag containing the MusicBrainz Identifiers (MBIDs) for the release artists.'),
     ),
     TagVar(
         'musicbrainz_albumid',
         shortdesc=N_('MusicBrainz Release Id'),
+        longdesc=N_('The MusicBrainz Identifier (MBID) for the release.'),
     ),
     TagVar(
         'musicbrainz_artistid',
         shortdesc=N_('MusicBrainz Artist Id'),
+        longdesc=N_('A multi-value tag containing the MusicBrainz Identifiers (MBIDs) for the track artists.'),
     ),
     TagVar(
         'musicbrainz_discid',
         shortdesc=N_('MusicBrainz Disc Id'),
+        longdesc=N_(
+            'The Disc ID is the code number which MusicBrainz uses to link a physical CD to a release listing. '
+            'This is based on the table of contents (TOC) information read from the disc. This tag contains the '
+            'Disc ID if the album information was retrieved using `Tools > Lookup CD` from the menu.'
+        ),
     ),
     TagVar(
         'musicbrainz_discids',
         shortdesc=N_('FIXME:musicbrainz_discids'),
+        longdesc=N_(
+            'A multi-value variable containing a list of all of the disc ids attached to the selected release. '
+            'The list provided for each medium only includes the disc ids attached to that medium. For example, '
+            'the list provided for Disc 1 of a three CD set will not include the disc ids attached to discs 2 '
+            'and 3 of the set.'
+        ),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'musicbrainz_originalalbumid',
         shortdesc=N_('MusicBrainz Original Release Id'),
```

Comment:
For those, I wonder if something likes:

`Original Release MBID`
`Original Artist MBID`
`Recording MBID`

wouldn't be better (there are shorter).
But it will change order, not sure if that's an issue.

@rdswift @phw what do you think?